tests/test_hdfs.py

```diff
@@ -269,13 +268,13 @@ def test_supervise():
         stop_max_delay=600*1000,
         retry_on_result=lambda res: not res)
     def wait_job_present(present):
-        svc = shakedown.get_service(JOB_SERVICE_NAME)
+        svc = shakedown.get_service(job_service_name)
```

Comment:
Nit: I would prefer to have `job_service_name` be a parameter for this inner function, or move the declaration of the `job_service_name` to before this function to make the intent clearer.

Reply:
fixed